lib/model/binding.dart

```diff
@@ -310,6 +314,19 @@ class PackageInfo {
   });
 }
 
+// Pigeon generates methods under `@EventChannelApi` annotated classes
+// in global scope of the generated file. This is a helper class to
+// namespace the notification related Pigeon API under a single class.
+class NotificationPigeonApi {
+  final _notifInteractionHost = notif_pigeon.NotificationHostApi();
```

Comment:
Is `_notifInteractionHost` the best name for this? How about `_hostApi`? We might add more to `NotificationHostApi` that's not about interacting with notifications (such as a method to query the current notification permissions). Also, `NotificationHostApi` isn't the only code that's about interacting with notifications; `notif_pigeon.notificationTapEvents` is too.